package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "ic_rails_phong_bliss",
+  "private": true,
```

Comment:
I recommend adding the "engine" key that contains the compatible `node` versions, but can also contain `npm` or `yarn` versions :)